recipes/cgns/all/patches/fix_static_or_shared.patch

```diff
@@ -0,0 +1,22 @@
+Enforces that either static or dynamic libs are build
```

Comment:
Looking at CMakeLists, this patch, and build logs, it's not true. This patch avoids to install static lib, but does not prevent to build both static & shared when `CGNS_BUILD_SHARED ON`